### ISSUE TITLE
[WP 4.7] Remove a11y inappropriate content from admin screen headings

### ISSUE THREAD

**Issue by JDGrimes** [WordPoints/wordpoints#578]
See https://core.trac.wordpress.org/changeset/39535. We probably need to do this on the modules screen. Possibly others. Because the CSS was committed in [WP@38983](https://core.trac.wordpress.org/changeset/38983) as part of 4.7, we can go ahead and pursue this in 2.3.0, even though WordPress won't fully fix all of its screens until 4.8.

**Comment by JDGrimes** [comment]
Yep, the extensions screen is the only one that has anything else in the heading.

Before:

![screenshot-2017-11-14 wordpoints extensions wordpress develop wordpress](https://user-images.githubusercontent.com/4005415/32784116-e93d1690-c91b-11e7-8bab-40b0619beb7f.png)

And with search:

![screenshot-2017-11-14 wordpoints extensions wordpress develop wordpress 1](https://user-images.githubusercontent.com/4005415/32784124-ee983e58-c91b-11e7-9d70-de8e790f7ed3.png)

After:

![screenshot-2017-11-14 wordpoints extensions wordpress develop wordpress 1](https://user-images.githubusercontent.com/4005415/32784271-5c706400-c91c-11e7-8c9e-4cd7c8b226b5.png)

And with search:

![screenshot-2017-11-14 wordpoints extensions wordpress develop wordpress](https://user-images.githubusercontent.com/4005415/32784285-61d3c5e0-c91c-11e7-8fe1-1ae0610f4281.png)

Basically, the two are visually identical, although the search text may have moved ever so slightly.
